Example/package.json

```diff
@@ -10,18 +10,20 @@
     "lint": "eslint ."
   },
   "dependencies": {
+    "@react-native/metro-config": "^0.72.12",
```

Comment:
why is this added?

Reply:
Required to run example app after upgrading RN version